contracts/verifiers/ProposalTemplates.sol

```diff
@@ -41,6 +36,8 @@ contract ProposalTemplates is Initializable, IProposalVerifier, Ownable, Version
     // templates library
     mapping(uint256 => ProposalTemplate) proposalTemplates; // proposal type => ProposalTemplate
 
+    constructor() Ownable(msg.sender) {}
```

Comment:
Using initialize() method would allows use deploy the contract as upgradable. Are we sure we don't want it for this contract?